back/db/schema.rb

```diff
@@ -54,6 +54,15 @@
     t.index ["rgt"], name: "index_admin_publications_on_rgt"
   end
 
+  create_table "analyses_tags", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name", null: false
+    t.uuid "analysis_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["analysis_id", "name"], name: "index_analyses_tags_on_analysis_id_and_name", unique: true
+    t.index ["analysis_id"], name: "index_analyses_tags_on_analysis_id"
+  end
+
```

Comment:
The `schema.rb` file still contains your old migration with the wrong table name.

Whenever you do rake db:migrate, rails will update this file. We use this in e.g. `db:reset` or tenant creation, because it's a lot faster and there is less chance of mistakes compared to running all migrations consecutively.

Since you've run db:migrate with the old and the new migration, the old one is there as well. You can either remove this manually (don't forget the corresponding `add_foreign_key` further down), or let rails regenerate it by overwriting it with the current file on master, doing `db:reset` and then running `db:migrate` again. The latter is probably the safer option.

Reply:
Ah, yes, missed that one 👍